src/main/frontend/components/block.cljs

```diff
@@ -2791,11 +2791,11 @@
            [:span.opacity-60.text-sm.ml-2.results-count
             (str (count transformed-query-result) " results")]]
            ;;insert an "edit" button in the query view
-           [:a.opacity-70.hover:opacity-100.svg-small.inline 
-            {:on-mouse-down (fn [e]
-                              (util/stop e)
-                              (editor-handler/edit-block! current-block :max (:block/uuid current-block)))}
-            svg/edit]]
+           (if (not built-in?) [:a.opacity-70.hover:opacity-100.svg-small.inline
```

Comment:
Use `when-not` instead of `if` because we only want to show the edit button when it's not a built-in query.
Also, it'll be great if we specify the hiccup view in a new line because it has multiple lines.